mercury/can_sensors.py

```diff
@@ -0,0 +1,17 @@
+TEMPERATURE = 1
+
+ACCELERATION_X = 2
+ACCELERATION_Y = 3
+ACCELERATION_Z = 4
+
+WHEEL_SPEED_FL = 5
+WHEEL_SPEED_FR = 6
+WHEEL_SPEED_BL = 7
+WHEEL_SPEED_BR = 8
+
+SUSPENSION_FL = 9
+SUSPENSION_FR = 10
+SUSPENSION_BL = 11
+SUSPENSION_BR = 12
+
+CURRENT_FUEL_LEVEL = 13
```

Comment:
I think we'll actually need a map the other direction, to have a number as the input (since it comes from the CAN ID field, and to map it to the proper sensor. I still am not sure how this will work with writing to the DB if these are separate, since for example, the model for wheel speed has all four sensors in a single row. We might need to allow nullable values and just write 4 separate rows for the "WheelSpeedSensor" unless we think of a better solution.

Reply:
From my understanding, you assign a number for each sensor (like TEMP =1) and then in the CAN message you check if the data is of type 1 (TEMP =1) to know if the data package is about temperature? Please rectify my understanding

Reply:
What you describe (as I understand it now) is exactly what I have here, but I think I need the reversed mapping, since I'll know the number, and need to know the sensor it relates to. I think I have an idea of what to do.